FreeRTOS_IP.c

```diff
@@ -2136,21 +2138,12 @@ static eFrameProcessingResult_t prvProcessIPPacket( IPPacket_t * pxIPPacket,
                          * went wrong because it will not be able to validate what it
                          * receives. */
                         #if ( ipconfigREPLY_TO_INCOMING_PINGS == 1 ) || ( ipconfigSUPPORT_OUTGOING_PINGS == 1 )
-                            if( pxNetworkBuffer->xDataLength >= sizeof( ICMPPacket_t ) )
                             {
-                                /* Map the buffer onto a ICMP-Packet struct to easily access the
-                                 * fields of ICMP packet. */
-                                ICMPPacket_t * pxICMPPacket = ipCAST_PTR_TO_TYPE_PTR( ICMPPacket_t, pxNetworkBuffer->pucEthernetBuffer );
-
                                 if( pxIPHeader->ulDestinationIPAddress == *ipLOCAL_IP_ADDRESS_POINTER )
```

Comment:
@htibosch I think you'll need this in your ipv6 branch

Reply:
`ipLOCAL_IP_ADDRESS_POINTER` only exists in the IPv4 branch.

Reply:
If I see it right both unicast and broadcast packets arrive here.
But I doubt that broadcast pings (`ping -b 255.255.255.255`) are handled well.